pkgs/applications/graphics/wings/default.nix

```diff
@@ -41,15 +65,17 @@ stdenv.mkDerivation rec {
     ${erlang}/bin/erl \
       -pa $out/lib/wings-${version}/ebin -run wings_start start_halt "$@"
     EOF
+
     chmod +x $out/bin/wings
+    runHook postInstall
   '';
 
   meta = {
```

Comment:
```suggestion
  meta = with lib; {
```

Reply:
Absolutely not. `with lib;` is an anti-pattern.

Reply:
why do you think that?

Reply:
What makes you think otherwise? It trips static analysis, my LSP and possibly any maintainer.

Reply:
I don't use LSP, so I can't relate. It looks a lot cleaner with lib;-ing the meta tag. It's also used pretty widely this way. I don't see an issue with it apart from a cleaner definition.

Reply:
Your personal preferences have no place in a technical review.

Reply:
Not my personal preference. Actively used practice. I recommend you just try doing a `rg "with lib;"` and see where it's used. It's actively used on meta tags. It's kind of childish to just abandon a discussion with a dead-kill statement and closing as resolved. Definitely not going to interact with you any longer.

Reply:
I'd like to remind you that `with lib;`, although common, is now a discouraged practice. "It looks cleaner" and "I don't use a LSP so can't relate" are both indicators of personal preference and I do not feel the need to entertain either of those. Additionally, this is a PR to update, format and migrate a package. "Add `with lib;` here it's cleaner!" is fully substential, and in my opinion, fully out of place.

If you are willing to understand why I'm choosing not to add it, see the issue above and [this comment](https://github.com/NixOS/nixpkgs/issues/292468#issuecomment-2016454242). Seeing we have nothing else to discuss, I will also not be interacting with you further.

Have a good day.

Reply:
You too. And thanks for your work.